websites/V/Void Bots/dist/metadata.json

```diff
@@ -0,0 +1,54 @@
+{
+  "author": {
+    "name": "DanPlayz",
+    "id": "209796601357533184"
+  },
+  "service": "Void Bots",
+  "description": {
+    "en": "Void Bots was designed to promote developers and their bots. We strive to create a friendly and inviting environment where developers can work together!"
+  },
+  "url": [
+    "voidbots.net",
+    "beta.voidbots.net",
+    "docs.voidbots.net"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/JZWaUA6.png",
+  "thumbnail": "https://i.imgur.com/eQfwKHr.png",
```

Comment:
try and get a better version of the logo (less pixelated?) which maybe has transparence outside of the circle?

Reply:
The logo that we provided does have transparency. (The image below is the 512x512 logo but downloaded and opened in an editor.) The pixelation you see is most likely from how zoomed in the default view of how most browsers show it. ![](https://discord.mx/u1mRZ84TwS.png)